content/en/docs/faq/terminology.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "TLS Terminology"
+linkTitle: "TLS Terminology"
+weight: 50
+type: "docs"
+---
+
+With TLS being so widely deployed, terminology can sometimes get confused or be used to mean different things, and that reality
+combined with the complexity of TLS can lead to serious misunderstandings and confusion.
+
+### What does "publicly trusted" mean?
+
+Broadly speaking, a "publicly trusted" certificate is one that you can use on the public internet and expect
+that most reasonably up-to-date computers will be able to verify it using their system trust store.
+
+There isn't a single standard trust store containing certs which are "publicly trusted", but generally most
+of the commonly seen trust stores are similar. An example would be [Mozilla's CA Certificate Program](https://wiki.mozilla.org/CA).
+
+### What does "self-signed" mean? Is my CA self-signed?
+
+Self-signed means exactly what it says; a certificate is self-signed if it is signed by its own private key.
+
+Self-signed is a commonly confused term, however, and is very frequently misused to mean "not publicly trusted". We tend to use terms
+like "private PKI" to denote the situation where an organization might have their own internal CA certificates which wouldn't
+be trusted outside of the organization.
+
+As an example, there are _many_ self-signed certificates in [Mozilla's CA Certificate Program](https://wiki.mozilla.org/CA), but
+all of those certificates would usually be described as "publicly trusted".
+
+Your certificate is self-signed only if it's signed with its own key.
+
+### What's the difference between "root", "intermediate", and "leaf" certificates?
+
+cert-manager uses the following definitions:
+
+#### Root Certificates
+
+Roots are self-signed certificates and almost always marked as CA certificates. They're usually not sent over the wire
+during a TLS handshake because they need to be explicitly trusted in order to be validated.
+
+Roots are sometimes defined as "CA certificates which are explicitly trusted" - which can include certificates which
+aren't self-signed. cert-manager doesn't use this definition.
+
+Changing trust stores to include new roots or remove old ones is a non-trivial task which can take months or years for publicly
+trusted roots. For this reason, roots are usually issued with very long lifetimes - often on the order of decades.
```

Comment:
suiggestion: we could mention the case of the [Let's Encrypt DST root CA X3](https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/) intermediate which expired in Sept 2021 and broke many old Android phones that were not being updated anymore; these phones cannot access some website due to this

A nice example of "long lifetime" that still broke things 😅

Reply:
I think Let's Encrypt's chains are super interesting and might be relevant somewhere in this doc because it's an example of an explicitly trusted _intermediate_ certificate in public trust stores.

I think I'd prefer not to add the example here; I think this issue shows the need for trust stores to be kept up to date, but that's kinda tangential to what this terminology guide is trying to achieve. That's more of an example for a TLS best-practices guide IMO.